test/lib/QuarkBuilderTest.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+import {Accounts} from "../../src/builder/Accounts.sol";
+import {PaymentInfo} from "../../src/builder/PaymentInfo.sol";
+import {QuarkBuilder} from "../../src/builder/QuarkBuilder.sol";
+
+contract QuarkBuilderTest {
+    address constant USDC_1 = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address constant USDC_8453 = 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913;
+
+    address constant ETH_USD_PRICE_FEED_1 = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address constant ETH_USD_PRICE_FEED_8453 = 0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70;
+
+    /**
+     *
+     * Fixture Functions
+     *
+     * @dev to avoid variable shadowing warnings and to provide a visual signifier when
+     * a function call is used to mock some data, we suffix all of our fixture-generating
+     * functions with a single underscore, like so: transferIntent_(...).
+     */
+    function transferUsdc_(uint256 chainId, uint256 amount, address recipient, uint256 blockTimestamp)
+        internal
+        pure
+        returns (QuarkBuilder.TransferIntent memory)
+    {
+        return QuarkBuilder.TransferIntent({
+            chainId: chainId,
+            sender: address(0xa11ce),
+            recipient: recipient,
+            amount: amount,
+            assetSymbol: "USDC",
+            blockTimestamp: blockTimestamp
+        });
+    }
```

Comment:
This should live in `QuarkBuilderTransfer.t.sol`

Reply:
moved